src/ts/components/account/LoginFooter.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import t from '../../services/i18n'
+import { Footer } from '../basic-components'
+import { colorSchemes } from '../styles/themes'
+import { IAppState } from '../../background/store/all'
+import { connect } from 'react-redux'
+
+const LoginFooter = (props) => {
+  const styles = {
+    loginFooter: {
+      width: '40%',
+      height: 25,
+      fontSize: 11,
+      margin: 'auto',
+      display: 'flex',
+      justifyContent: 'space-around'
+    },
+    link: {
+      color: colorSchemes[props.settings.color].backgroundColor
```

Comment:
can you move link color in basic component ? then you dont need to connect to store here

Reply:
Would that affect the Speckle 2019 color?

Reply:
not sure if we can targe child element css in styled component

Reply:
Added StyledLink to basic-components, so that the LoginFooter can use that and avoid connecting to store.